stories/CodeView.stories.tsx

```diff
@@ -56,7 +56,12 @@ const render = ({
     version: createInternalVersion(fakeVersion),
     ...moreProps,
   };
-  return renderWithStoreAndRouter(<CodeView {...props} />, store);
+  return renderWithStoreAndRouter(
+    <div className="CodeView-panel">
```

Comment:
Why is this needed?

Reply:
`CodeView` no longer renders its own border. This makes it hard to tell where the title of the story ends and the component content begins. I added a border around the story content which highlights the component's container.